docs/it/trust_marks_cie.rst

```diff
@@ -4,12 +4,35 @@
 Trust Mark per CIE
 ++++++++++++++++++
 
-I Trust Mark sono JSON Web Token (JWT) firmati che rappresentano una dichiarazione di conformità ad un insieme ben definito di requisiti di fiducia e/o interoperabilità, oppure ad un accordo fra le parti coinvolte nella Federazione e vengono emessi da entità accreditate, principalmente durante il processo di Onboarding. Lo scopo principale è convogliare alcune informazioni non propriamente richieste dal protocollo stesso OIDC, ma che potrebbero essere utili all’interno della Federazione. Tipici esempi includono il codice di identificazione nazionale dell’entità, contatti istituzionali e caratteristiche supportate disponibili all’interno dell’ecosistema CIE. Dati aggiuntivi possono essere aggiunti dall’emittente e devono essere ben compresi.
+TBD esempi non normativi
 
-Nello scenario CIE, un Trust Mark viene firmato da **MinInterno** (TA) o da un’entità accreditata (es. entità intermedie (SA) o Autorità Attributo (AA) che giocano il ruolo di entità Risorse Protette OAuth - un’entità che agisce come AA all’interno del sistema della Federazione CIE, può essere visto come un tipo di entità Risorsa Protetta OAuth, in accordo a `OIDC-FED#Section.4.5`_) e DEVONO essere incluse nella richiesta (attributo) dei Trust Mark della configurazione di entità delle foglie (RP e OP) e di intermediari (SA). La presenza di un Trust Mark è richiesta prima di iniziare una scoperta di metadati (vedere sezione), altrimenti la federazione può essere soddisfatta da aggressori che cercano di propagare attacchi. to propagate attacks.
+Trust Mark Attribute Authority CIE
```

Comment:
@sp640919 , rimuoverei questa sezione:
- Non esistono AA CIE
- Andrebbero nell'Allegato tecnico AA

@peppelinux , valutiamo se fare una sezione ad hoc sulle AA con lo stesso spirito di questo documento. In caso questa informazione la riportiamo lì.

Reply:
si ma non credo che ce la faremo prima che la AgID pubblichi AT OAS3 per OIDC/OAuth2

Reply:
rimossa la sezione 
Trust Mark Attribute Authority CIE